apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/(sidebar)/insight/layout.tsx

```diff
@@ -0,0 +1,113 @@
+import { redirect } from "next/navigation";
+import { getProject } from "@/api/projects";
+import { TabPathLinks } from "@/components/ui/tabs";
+import { UnderlineLink } from "@/components/ui/UnderlineLink";
+import { FooterLinksSection } from "../components/footer/FooterLinksSection";
+
+export default async function Layout(props: {
+  params: Promise<{
+    team_slug: string;
+    project_slug: string;
+  }>;
+  children: React.ReactNode;
+}) {
+  const params = await props.params;
+  const project = await getProject(params.team_slug, params.project_slug);
+
+  if (!project) {
+    redirect(`/team/${params.team_slug}`);
+  }
+
+  const insightLayoutPath = `/team/${params.team_slug}/${params.project_slug}/insight`;
+
+  return (
+    <div className="flex grow flex-col">
+      <div className="pt-4 lg:pt-6">
+        <div className="container max-w-7xl">
+          <h1 className="mb-1 font-semibold text-2xl tracking-tight lg:text-3xl">
+            Insight
+          </h1>
+          <p className="max-w-3xl text-muted-foreground text-sm leading-relaxed">
+            APIs to retrieve blockchain data from any EVM chain, enrich it with
+            metadata, and transform it using custom logic.{" "}
+            <UnderlineLink
+              href="https://portal.thirdweb.com/insight"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              Learn more
+            </UnderlineLink>
+          </p>
+        </div>
+
+        <div className="h-4" />
+        <TabPathLinks
```

Comment:
Remove tabs - not required, since there's only a single page. We can add this when we have multiple pages

Reply:
This will also fix the state where FTUX is shown under "analytics" 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/fa38fef4-2791-420f-a836-f71b5c971165.png)